config/routes.rb

```diff
@@ -1,6 +1,24 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  mount Rswag::Ui::Engine => '/api-docs'
+  mount Rswag::Api::Engine => '/api-docs'
+  get 'current_user/index'
+  get 'current_user/show'
+  devise_for :users, controllers: {
+    sessions: 'users/sessions',
+    registrations: 'users/registrations',
+  }
+
+  resources :current_user, only: [:index, :show]
 
   # Defines the root path route ("/")
-  # root "articles#index"
+  namespace :api do
+    namespace :v1 do
+      resources :users do
+        resources :appointments
+      end
+      resources :technicians do
+        resources :appointments
```

Comment:
- [ ]  Nice routes and nice use of the `#resources` method. However, you are generating more routes than needed.

## Explanation

By using the `only:` keyword parameter, you can generate only the routes that you are using.

[Look](https://launchschool.com/books/demystifying_rails/read/routes_and_resources):

![https://user-images.githubusercontent.com/57421823/201431229-f7861d5e-3462-4897-b6fa-1242581bf06f.png](https://user-images.githubusercontent.com/57421823/201431229-f7861d5e-3462-4897-b6fa-1242581bf06f.png)

## Example

```ruby
resources :vehicles, only: [:index, :create, :show, :destroy]

```

That way, your routes will be more consistent, and only the ones needed will be generated💪

Reply:
Thank you for your comment @Meltrust . We have it in mind to continue working on the project and implementing the other routes, that is why we used the resources without limiting the routes